.github/scripts/run_consensus.sh

```diff
@@ -7,11 +7,8 @@ mkdir -p docker/data/
 # source account ids
 source docker/env
 
-# Generate chainspec and populate comittee keystores
```

Comment:
I would leave a word or two telling what these two long lines do. 

Sorry for my bias, but I am very lame at bash and I strongly prefer scripts accompanied with human explanation :man_shrugging: